oada-srvc-docker-config.js

```diff
@@ -37,7 +37,7 @@ module.exports = {
     collections: {
       users: {
         name: 'users',
-        indexes: ['username', { name: 'oadaid', sparse: true }],
```

Comment:
Why all these minor space changes?

Reply:
I assume I'm the first to touch it after prettier was turned on.